addons/point_of_sale/static/tests/pos/tours/utils/payment_screen_util.js

```diff
@@ -378,8 +378,8 @@ export function clickPartnerButton() {
     ];
 }
 
-export function clickCustomer(name) {
-    return [...PartnerList.searchCustomerValue(name), PartnerList.clickPartner(name)];
+export function clickCustomer(name, pressEnter = true) {
```

Comment:
could you put the default value for `pressEnter = false` (as it's the case for the function `searchCustomerValue
```suggestion
export function clickCustomer(name, pressEnter = false) {
```